### ISSUE TITLE
Subscription form: Move label out of email input

### ISSUE THREAD

**Issue by MichaelArestad** [Automattic/wp-calypso#29189]
Right now, the label for the Subscription form email input is in the input itself. This is not great. We should move the label to be a proper label and instead of a placeholder in the email input.

Here is what it looks like now:

![image](https://user-images.githubusercontent.com/1123119/49599254-c6e4ba00-f93d-11e8-8ef8-08ac78974878.png)

Instead, it should behave more like the Contact Form shown here:

![image](https://user-images.githubusercontent.com/1123119/49599301-e11e9800-f93d-11e8-9ec3-7cd22b82c544.png)

This will possibly break some styles, but will be an improvement for accessibility.
